### PR TITLE
Join neighborhood

### PR DESCRIPTION
**Pull request by nina-dk**

### CODE REVIEW

File: apps/backend/src/services/neighborhoodServices.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ const parseCreateNeighborhoodData = async (object: unknown): Promise<CreateNeigh
 /**
  * associates user with the neighborhood in the db
  * throws error if userId or neighborhoodId invalid
- * throws error if user already associates with the neighborhood
+ * throws error if user already associated with the neighborhood
```
Comment (radu-constantin) [inline apps/backend/src/services/neighborhoodServices.ts:196]: How do you even see these? :))

File: apps/frontend/src/pages/SingleNeighborhoodPage/SingleNeighborhoodPage.tsx [hunk 2]
```diff
@@ -39,7 +39,17 @@ export async function action({ params, request }: ActionFunctionArgs) {
   const requestData = Object.fromEntries(formData) as unknown as RequestData;
   requestData.neighborhoodId = Number(params.id);
 
-  const response = await requestServices.createRequest(requestData);
+  const intent = formData.get('intent');
```
Comment (radu-constantin) [inline apps/frontend/src/pages/SingleNeighborhoodPage/SingleNeighborhoodPage.tsx:42]: Good thinking with this!

**Review by radu-constantin — APPROVED** [review]
Everything works fine. Great job!

File: apps/frontend/src/pages/SingleNeighborhoodPage/SingleNeighborhoodPage.tsx (hunk 2 above)
Comment (nina-dk) [reply apps/frontend/src/pages/SingleNeighborhoodPage/SingleNeighborhoodPage.tsx:42]: Thanks, I got it from the react-router docs

**Pull request merged**